files/zh-cn/web/html/element/span/index.md

```diff
@@ -1,55 +1,116 @@
 ---
-title: <span>
+title: <span>：内容行级元素
 slug: Web/HTML/Element/span
+l10n:
+  sourceCommit: fdd3ac5598c3ddceb71e59949b003936ae99f647
 ---
 
 {{HTMLSidebar}}
 
-**HTML `<span>`** 元素是短语内容的通用行内容器，并没有任何特殊语义。可以使用它来编组元素以达到某种样式意图（通过使用类或者 Id 属性），或者这些元素有着共同的属性，比如**lang**。应该在没有其他合适的语义元素时才使用它。`<span>` 与 {{HTMLElement("div")}} 元素很相似，但 {{HTMLElement("div")}} 是一个 [块元素](/zh-CN/docs/HTML/Block-level_elements) 而 `<span>` 则是 [行内元素](/zh-CN/docs/HTML/Inline_elements).
+**`<span>`** [HTML](/zh-CN/docs/Web/HTML) 元素是一个通用的内联容器，本身不具备特殊含义。它常用于将元素集合起来以便进行样式化处理（通过使用 [`class`](/zh-CN/docs/Web/HTML/Global_attributes#class) 或 [`id`](/zh-CN/docs/Web/HTML/Global_attributes#id) 属性），或者因为它们共享属性值，例如 [`lang`](/zh-CN/docs/Web/HTML/Global_attributes#lang) 属性。该元素仅应在无其他合适语义元素时使用。`<span>` 与 {{HTMLElement("div")}} 元素非常相似，但是 {{HTMLElement("div")}} 是一个[块级内容](/zh-CN/docs/Glossary/Block-level_content)，而 `<span>` 是一个[行级内容](/zh-CN/docs/Glossary/Inline-level_content)。
```

Comment:
1. 似乎最前面原语序 HTML `<span>` 元素更加顺一点？
2. 集合元素 => 分组（后者好像更常用更好理解？）
3. lang 的例如，建议加括弧或者破折，让层次更为清晰？

```suggestion
**`<span>`** [HTML](/zh-CN/docs/Web/HTML) 元素是一个通用的内联容器，本身不具备特殊含义。它可被用于将元素分组，以便于进行样式化处理（通过使用 [`class`](/zh-CN/docs/Web/HTML/Global_attributes#class) 或 [`id`](/zh-CN/docs/Web/HTML/Global_attributes#id) 属性），或是因为它们共享属性值（例如 [`lang`](/zh-CN/docs/Web/HTML/Global_attributes#lang) 属性）。该元素仅应在无其他合适语义元素时使用。`<span>` 与 {{HTMLElement("div")}} 元素非常相似，但是 {{HTMLElement("div")}} 是一个[块级内容](/zh-CN/docs/Glossary/Block-level_content)，而 `<span>` 是一个[行级内容](/zh-CN/docs/Glossary/Inline-level_content)。
```

感觉原文 `It can be used to group elements for styling purposes (using the [class](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#class) or [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#id) attributes), or because they share attribute values, such as [lang](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#lang).` 和中文语序差太多了，可能得做一下调整。

Reply:
文档中有写元素集合的 也有写元素分组的 可能是历史遗留问题了。@yin1999 你能帮忙看看吗

Reply:
元素集合和元素分组应该都没什么问题，groupping也只是一个普通动词叭，不过感觉，可能如果用分组更加容易get the point（集合更多用作名词了）？ ：）

Reply:
大致赞同 @PassionPenguin 的修改，需要把“inline container”的翻译改成“行级容器”，然后第二处的“因为它们共享属性”，这个翻译有点“直译”的感觉：接上一句，以便于添加样式，或共享属性值。